src/apps/frontend/pages/profile/settings/index.tsx

```diff
@@ -18,7 +18,7 @@ const ProfileSettings = () => {
   const navigate = useNavigate();
 
   const [isDeleteAccountModalOpen, setIsDeleteAccountModalOpen] =
-    useState(false);
+    React.useState(false);
```

Comment:
This sounds like a bandaid solution. We should look into the right way to import, using `React.` seems undesirable.